src/js/modules/infragistics.ui.tilemanager.js

```diff
@@ -1677,33 +2330,45 @@
         },
         maximizedTile: function () {
             /* Returns the maximized tile or null if such is not present.
-               returnType="object|null" Returns the maximized tile or null if such is not present.
+            ```
+                $(".selector").igTileManager( "maximizedTile");
+            ```
+            returnType="object|null" Returns the maximized tile or null if such is not present.
             */
             return this._options.$maximizedTile || null;
         },
         minimizedTiles: function () {
             /* Returns an array with the tiles in minimized state or null if such are not present.
+            ```
+                $(".selector").igTileManager( "minimizedTiles");
+            ```
                returnType="object|null" Returns an array with the tiles in minimized state or null if such are not present.
             */
             var minimizedTiles = this._options.$tiles.not(this._options.$maximizedTile);
             return minimizedTiles.length > 0 ? minimizedTiles : null;
         },
         splitter: function () {
             /* Returns the splitter associated with this tile manager or
-               null if the tile manager was instantiated with maximizedTileIndex.
-               returnType="object|null" Returns the splitter associated with this tile manager or
-               null if the tile manager was instantiated with maximizedTileIndex.
+                null if the tile manager was instantiated with maximizedTileIndex.
+            ```
+                $('.selector').igTileManager( "splitter")
```

Comment:
There is a space in all code snippets before the method name that is being called.